### ISSUE TITLE
Disable invalid times for time-picker with range

### ISSUE THREAD

**Issue by Darent98** [Vuepic/vue-datepicker#459]
When selecting time ranges with time-picker and range active, there is a validation so that the second time cannot be before the first time and vice versa. Still you can click on the button to select an invalid time, but the time is not selected (as it should be).

It would be nice if the times, that are invalid, are disabled to show the user they cannot select this time. 
Then it should look like you set a min-time (not allowed times are grey and cannot be clicked on).
The first time fields (hours and minutes) will have all times disabled after the selection in the second time fields and the second time fields will have all times disabled before the first time fields.

I tried using the min-time and max-time, but they disable the times for both time fields and cannot be used for this scenario.

Here is a link to an example, where i also set the min-time to show how it could look: https://stackblitz.com/edit/vuepic-vue-datepicker-jsfpqj?file=src%2Fcomponents%2FPlayground.vue

**Comment by Darent98** [comment]
Hi,

Thank you for your attempt to implement the suggested feature. Unfortunately, it doesn't function as anticipated. I'm not sure if this is due to a configuration issue on my end or a possible misunderstanding regarding the requested feature.

The problem I encountered is that, in a range time-picker, the invalid times aren't displayed as disabled. To help clarify, I've provided a few screenshots illustrating the desired behavior:

1. The first picture displays the selected start and end times.
![base](https://github.com/Vuepic/vue-datepicker/assets/84193036/ded372af-4f5c-42ee-baef-32bcd2668c8b)

2. The second picture demonstrates how the time-picker for the start time should appear, with times from 10:24 onwards being disabled.
![start_time_selection](https://github.com/Vuepic/vue-datepicker/assets/84193036/0ccb61fe-57ea-41ed-ae5d-b56e3a0f1cc4)

3. The third picture depicts the end time-picker, where times before 8:24 should be disabled.
![end_time_selection](https://github.com/Vuepic/vue-datepicker/assets/84193036/20ecbbf6-075c-4d8b-9a2b-19a06d6e6486)

If these functionalities are present in your most recent update, could you please provide a minimal working example? If they're not, kindly consider reopening this ticket.

Thank you.

**Comment by Jasenkoo** [comment]
@Darent98 please take a look at this [exapple](stackblitz.com/editįvuepic-vue-datepicker-9cpu3t?file=src%2Fcomponents%2FPlayground.vue)

There is another way to disable specific times in the overlay using filters, but it looks like I missed to port it, will be corrected.

**Comment by Darent98** [comment]
@Jasenkoo in the example you provided, the link seems to be broken for me. I've found the corrected link here:  [corrected link](https://stackblitz.com/edit/vuepic-vue-datepicker-9cpu3t?file=src%2Fcomponents%2FPlayground.vue). I noticed that even though all minute values are disabled, hour '15' is not grayed out. Additionally, this example doesn't demonstrate its use within a range time-picker.

**Comment by Jasenkoo** [comment]
@Darent98 You are right, I'll have to make an adjustment to the logic, and will include it in the next release.
